extensions/amp-video/0.1/amp-video.js

```diff
@@ -222,9 +225,7 @@ export class AmpVideo extends AMP.BaseElement {
 
     this.video_ = element.ownerDocument.createElement('video');
     // Manage video if the sources contain bitrate or amp-orig-src will be expanded to multiple bitrates.
-    if (
-      this.element.querySelector('source[data-bitrate], source[amp-orig-src]')
-    ) {
+    if (this.isManagedByBitrate_ && !this.isPlaceholderVideo_(this.video_)) {
```

Comment:
Are these changes required for this PR to work? If not, let's remove them for now

Reply:
This is because we don't want to manage the placeholder video, only the mediapool video. Not sure if removing this change is breaking, but it's the main point of this PR's idea to move the management from here to the `resetOnDomChange`